### DISCUSSION TITLE
LIST consumers and producers

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5929] [Q&A]
is there any easy way to list the consumers and producers connected to a cluster in strimzi?   not using any monitoring tools?    more looking for kubectl or command line util.

**Comment by scholzj** [comment]
Kubernetes do not know about your Kafka connections. So you cannot use `kubectl` for this. I'm not sure how much you can get from the Kafka tools => the consumer groups show some information about connections. But not sure if you can get something better from some other tool.
